src/GovUk.Education.ExploreEducationStatistics.Admin.Tests/Services/FootnoteServiceTests.cs

```diff
@@ -32,33 +34,742 @@ namespace GovUk.Education.ExploreEducationStatistics.Admin.Tests.Services
     public class FootnoteServiceTests
     {
         private readonly DataFixture _fixture = new();
-        
+
+        [Fact]
+        public async Task CreateFootnote_SubjectNotLinkedToRelease_ReturnsFootnoteSpecificationsAreInvalidValidationResult()
+        {
+            Release release = _fixture.DefaultStatsRelease().Generate();
```

Comment:
Again, here I'd argue that the concrete type should be used because it isn't **explicit** from the RHS. Have we agreed a coding standard for this?

I think always having either the LHS or the RHS being explicit is most readable. You don't need to inspect the methods on the RHS to see what their return type is. It's more discoverable and requires less detective work

Reply:
going to discuss in backend guild meeting

Reply:
See reply to other comment.